services/src/main/java/org/exoplatform/addons/gamification/service/configuration/ChallengeServiceImpl.java

```diff
@@ -22,9 +24,16 @@
 
   private SpaceService     spaceService;
 
-  public ChallengeServiceImpl(ChallengeStorage challengeStorage, SpaceService spaceService) {
+  private String     CREATORS_GROUP;
+
+  private static final String     CREATORS_GROUP_KEY             = "challenge.creator.group";
+
+
+  public ChallengeServiceImpl(ChallengeStorage challengeStorage, SpaceService spaceService, InitParams params) {
     this.challengeStorage = challengeStorage;
     this.spaceService = spaceService;
+    this.CREATORS_GROUP = params.getValueParam(CREATORS_GROUP_KEY).getValue();
```

Comment:
Please check for nullity here
```suggestion
    this.groupOfCreators = params.getValueParam(CREATORS_GROUP_KEY).getValue();
```